config.json

```diff
@@ -857,6 +857,22 @@
         "nugetId": "Xamarin.AndroidX.WebKit",
         "dependencyOnly": false
       },
+      {
+        "groupId": "androidx.window",
+        "artifactId": "window",
+        "version": "1.0.0-alpha01",
+        "nugetVersion": "1.0.0.0-alpha01",
+        "nugetId": "Xamarin.AndroidX.Window.Window",
```

Comment:
Looking at other examples like `webkit` and `wear`, it seems like this should not duplicate `window` and should just be `Xamarin.AndroidX.Window`.

Reply:
We always had discussion and there were 2 or 3 cases where this shortening burned me quite bad.

But I will shortent that one!